recipe/meta.yaml

```
@@ -13,10 +13,13 @@ source:
     - fix_dispatch_apply_auto.patch
 
 build:
-  number: 0
+  number: 1
   run_exports:
     - {{ pin_subpackage('pytorch', max_pin='x.x.x') }}
   skip: true  # [win]
+  skip: true  # [cuda_compiler_version=="10.2"]
+  skip: true  # [cuda_compiler_version=="11.0"]
+  skip: true  # [cuda_compiler_version=="11.1"]
```

Just so I understand, the goal is just to build with CUDA 11.2 and use CUDA Enhanced Compatibility for CUDA 11.2+?

Also we could [fuse these into a single `skip`]( https://github.com/conda-forge/mandrake-feedstock/blob/6b5ff9263188eaa9c316bdce3b418427bef9581c/recipe/meta.yaml#L14 )

```suggestion
  skip: true  # [cuda_compiler_version in ("10.2", "11.0", "11.1")]
```

---

yes you are correct.